numpy/distutils/exec_command.py

```diff
@@ -253,7 +253,13 @@ def _exec_command(command, use_shell=None, use_tee = None, **env):
     except EnvironmentError:
         # Return 127, as os.spawn*() and /bin/sh do
         return 127, ''
-    text, err = proc.communicate()
+    try:
+        text, err = proc.communicate()
```

Comment:
Is it not possible to tell Popen to not try to decode the input at all?

Reply:
That should have been disabled with `universal_newlines=False`, but apparently wasn't.